test_fastdupes.py

```diff
@@ -0,0 +1,12 @@
+#execute with pytest
+
+from fastdupes import find_dupes
+
+def test_common_prefix(tmpdir):
+    files = tmpdir.mkdir("files")
+    file1 = files.join("file1")
+    file2 = files.join("file2")
+    file1.write("0"*1000000 + "1")
+    file2.write("0"*1000000 + "2")
```

Comment:
It'd probably be better to either assert or use `max()` to ensure that this test will always use a value greater than `HEAD_SIZE`.